.gitlab-ci.yml

```diff
@@ -63,21 +63,21 @@ cargo-check 0 3:
   <<:                              *docker-cache-status
   script:
     - time cargo check --target $CARGO_TARGET --locked --no-default-features --verbose --color=always
-    - sccache --stop-server
+    - sccache -s
 
 cargo-check 1 3:
   stage:                           test
   <<:                              *docker-cache-status
   script:
     - time cargo check --target $CARGO_TARGET --locked --manifest-path util/io/Cargo.toml --no-default-features --verbose --color=always
-    - sccache --stop-server
+    - sccache -s
 
 cargo-check 2 3:
   stage:                           test
   <<:                              *docker-cache-status
   script:
     - time cargo check --target $CARGO_TARGET --locked --manifest-path util/io/Cargo.toml --features "mio" --verbose --color=always
-    - sccache --stop-server
+    - sccache -s
```

Comment:
What's wrong with `--stop-server`?

Reply:
i's  not work in darwin. and this is more correct

Reply:
-s show stats only. --stop-server - stop server and show stats

Reply:
```suggestion
    - sccache --show-stats
```

More readable imo!